dmx_client/dmx_client.py

```diff
@@ -17,13 +20,26 @@
 
 
 async def handler():
-	async with websockets.connect(f'ws://{SERVER_IP}:{SERVER_PORT}') as websocket:
+	async with websockets.connect('ws://{}:{}'.format(SERVER_IP, SERVER_PORT)) as websocket:
 		while True:
 			message = await websocket.recv()
 
 			dmx_values = message.split(',')
-			for index, value in enumerate(dmx_values):
-				dmx_device.set_data(index+1, value)
+			dmx_values = dmx_values[3*12:]
+			dmx_values = dmx_values[::-1]
+			# dmx_values.reverse()
+
+			dmx_values = [max(0, min(int(x), 255)) for x in dmx_values]
```

Comment:
co jeśli nie będzie intem - będzie gdzieś niepoprawne parsowanie ?

Reply:
wywali kod, ale to my wysyłamy sender'em z backendu, więc nie powinno być takiej sytuacji. Automatycznie wstanie więc na luzie :D